workshop/src/main/java/nl/jcore/java8workshop/Exercise002ParameterNamesDto.java

```diff
@@ -12,7 +10,7 @@
  * Maven goal "clean" and run the plugins compiler:compile
  * and compiler:testCompile.
  */
-@Immutable
+/* @Immutable */
```

Comment:
Because the @Immutable annotation used is an internal annotation used in the JDK for Project Nashorn (Javascript on the JVM). So I understand your intent, but the wrong implementation was used. But I agree, remove it, as it's easy to see that the class is immutable any way.